Menus/MainMenu/MainMenu.cs

```diff
@@ -97,11 +83,9 @@ public bool visible
 
 		public Transform rayOrigin { private get; set; }
 
-		public float hideDistance { get { return m_MainMenuUI.menuHeight; } }
-
 		public Bounds localBounds { get { return m_MainMenuUI.localBounds; } }
 
-		public bool hovering { get { return m_MainMenuUI.hovering; } }
+		public bool focus { get { return m_MainMenuUI.hovering; } }
```

Comment:
I thought of that, but I figured that "hovering" is what's happening here. It makes sense that a menu might abstractly be "focused" by a number of things, but in the case of the UI, hovering is what's happening.